package.json

```diff
@@ -16,7 +16,7 @@
   "author": "mesur.io <oss@mesur.io>",
   "license": "MIT",
   "dependencies": {
-    "@capacitor/core": "latest"
+    "@capacitor/core": "^2.4.7"
```

Comment:
I believe `^2.4.7` is also required for dev deps on @capacitor/android and @capacitor/ios

Reply:
thx, changed it